src/main/python/utils/s3_util_class.py

```diff
@@ -0,0 +1,84 @@
+import csv
```

Comment:
Can change name of the file to `s3_connector.py` . we can also add download functionality. So, the class cane be more generic - something like `class S3Connector` below.

Reply:
Sure